### PR TITLE
test(db): add missing function coverage

### PR DESCRIPTION
**Pull request by beeman**
Add focused tests for previously untested db helpers, metadata accessors, population utilities, sanitizers, schema parsing, and order calculators.

Cover setting list queries and supporting pure utilities in line with the surrounding Vitest structure.

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive test coverage for database account and wallet operations including creation order determination, network and settings management, data sanitization and validation, transaction signature schema verification, random identifier generation, and database reset functionality. New utility functions provide database table metadata discovery, individual and bulk record retrieval, and table management capabilities.

### CODE REVIEW

**Comment by beeman** [comment]
@coderabbitai review

File: packages/db/test/parse-strict.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      expect(result).toEqual({
+        count: 0,
+        enabled: false,
+        name: '',
+        nested: null,
+      })
+      expect(result).not.toHaveProperty('missing')
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/parse-strict.test.ts:55]: Added the required unexpected behavior section with console.log mocking and an all-undefined input edge case.

File: packages/db/test/create-db.test.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+      // ACT
+      const result = createDb({ name })
+
+      // ASSERT
+      expect(result).toBeInstanceOf(Database)
+      expect(result.name).toBe(name)
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/create-db.test.ts:42]: Added the required unexpected behavior section with console.log mocking and an empty-name edge-case assertion.

File: packages/db/test/reset.test.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        accounts: 0,
+        bookmarkAccounts: 0,
+        bookmarkTransactions: 0,
+        networks: 3,
+        settings: 2,
+        wallets: 0,
+      })
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/reset.test.ts:67]: Added the required unexpected behavior section with console.log mocking and an already-empty reset edge-case assertion.

File: packages/db/test/random-id.test.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+      // ACT
+      const result = randomId(8)
+
+      // ASSERT
+      expect(result).toHaveLength(8)
+      expect(result).toMatch(/^[A-Z0-9]+$/)
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/random-id.test.ts:51]: Added the required unexpected behavior section with console.log mocking and a zero-size id edge-case assertion.

File: packages/db/test/populate-networks.test.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+          name: 'Testnet',
+          type: 'solana:testnet',
+        },
+      ])
+      expect(result.every((network) => network.createdAt instanceof Date && network.updatedAt instanceof Date)).toBe(
+        true,
+      )
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/populate-networks.test.ts:71]: Added the required unexpected behavior section with console.log mocking and an assertion for omitted unconfigured networks.

File: packages/db/test/wallet-determine-name.test.ts [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      ]
+
+      // ACT
+      const result = walletDetermineName(items)
+
+      // ASSERT
+      expect(result).toBe('Wallet 4')
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/wallet-determine-name.test.ts:75]: Added the required unexpected behavior section with console.log mocking and a non-exact wallet-name edge-case assertion.

File: packages/db/test/populate.test.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+      // ASSERT
+      expect(result1.map((network) => network.id)).toEqual(['networkDevnet', 'networkLocalnet', 'networkTestnet'])
+      expect(result2.map((setting) => [setting.key, setting.value])).toEqual([
+        ['activeNetworkId', env('activeNetworkId')],
+        ['apiEndpoint', env('apiEndpoint')],
+      ])
+    })
+  })
+})
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/populate.test.ts:50]: Added the required unexpected behavior section with console.log mocking and a duplicate-populate rejection test.

**Comment by beeman** [comment]
@coderabbitai review

**Comment by beeman** [comment]
@coderabbitai review

File: packages/db/test/network-get-label.test.ts [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    })
+
+    afterEach(() => {
+      vi.restoreAllMocks()
+    })
+
+    it('should return an unknown label for an unsupported network type', () => {
+      // ARRANGE
+      expect.assertions(1)
+      const type = 'solana:custom' as NetworkType
```
[the remark this answers is not in the document]
Comment (beeman) [reply packages/db/test/network-get-label.test.ts]: Replaced the cast with the project-standard @ts-expect-error annotation for the intentional invalid NetworkType input.

**Comment by beeman** [comment]
@coderabbitai review
